instance-scheduler/main_test.go

```diff
@@ -187,6 +189,412 @@ func (m *mockIEC2InstancesAPI) StartInstances(ctx context.Context, params *ec2.S
 	return m.StartInstancesOutput, nil
 }
 
+type mockIRDSInstancesAPI struct {
+	DescribeDBInstancesOutput *rds.DescribeDBInstancesOutput
+	StartDBInstanceOutput     *rds.StartDBInstanceOutput
+	StopDBInstanceOutput      *rds.StopDBInstanceOutput
+}
+
+func (m *mockIRDSInstancesAPI) DescribeDBInstances(ctx context.Context, params *rds.DescribeDBInstancesInput, optFns ...func(*rds.Options)) (*rds.DescribeDBInstancesOutput, error) {
+	return m.DescribeDBInstancesOutput, nil
+}
+
+func (m *mockIRDSInstancesAPI) StopDBInstance(ctx context.Context, params *rds.StopDBInstanceInput, optFns ...func(*rds.Options)) (*rds.StopDBInstanceOutput, error) {
+	return m.StopDBInstanceOutput, nil
+}
+
+func (m *mockIRDSInstancesAPI) StartDBInstance(ctx context.Context, params *rds.StartDBInstanceInput, optFns ...func(*rds.Options)) (*rds.StartDBInstanceOutput, error) {
+	return m.StartDBInstanceOutput, nil
+}
+func TestStopStartTestRDSInstancesInMemberAccount(t *testing.T) {
+	tests := []struct {
+		testTitle     string
+		client        *mockIRDSInstancesAPI
+		action        string
+		expectedCount RDSInstanceCount
+	}{
+		{
+			testTitle: "RDS testing Stop action",
+			client: &mockIRDSInstancesAPI{
+				DescribeDBInstancesOutput: &rds.DescribeDBInstancesOutput{
+					DBInstances: []rdstype.DBInstance{
+						// RDS instance-scheduling = default, therefore schedule an instance, acted upon: 1
+						{
+							DBInstanceIdentifier: aws.String("test-database"),
+							TagList: []rdstype.Tag{
+								{
+									Key:   aws.String("instance-scheduling"),
+									Value: aws.String("default"),
+								},
+							},
+						},
+						// aws:autoscaling:groupName is set, therefore skip scheduling, skipped auto scaled: 1
```

Comment:
there is nothing to test, the autoscaling group is not included in the business logic